field.go

```diff
@@ -132,6 +132,10 @@ func OptFieldTypeInt(min, max int64) FieldOption {
 }
 
 func OptFieldTypeTime(timeQuantum TimeQuantum) FieldOption {
+	return OptFieldTypeTimeOptions(timeQuantum, false)
+}
+
+func OptFieldTypeTimeOptions(timeQuantum TimeQuantum, noStandardView bool) FieldOption {
```

Comment:
what do you think about changing the function signature of `OptFieldTypeTime` to be:
```
func OptFieldTypeTime(timeQuantum TimeQuantum, args ...bool) FieldOption
```
so that we don't have to introduce another Opt function? I believe technically that wouldn't break an existing implementation, and we could check for `len(args)` and use `args[0]` to represent `noStandardView`.

Reply:
Regardless, I think we should add a `TODO (2.0)` comment to clean this up. At that point, I feel like we should suppress the standard view by default, and possibly not support it at all; I'm not convinced that it's a useful view, and it can easily be supported by maintaining a separate `set` field.

Reply:
I guess my thinking is that we won't likely be adding anything else to this function until we get to 2.0. I realize that that may not be the case, but I would be ok taking that chance.

And I agree, it's definitely not a clean solution, but if feels a bit more contained than adding a new function that is almost the same thing. Again, I see this as a somewhat temporary solution until 2.0.

Reply:
updated